node/internal/parse_yarn_lock.js

```diff
@@ -26,16 +26,17 @@ function main() {
   //
   const entries = Object.keys(yarn.object).map(key => makeYarnEntry(key, yarn.object[key]));
 
-  // For all top-level folders in the node_modules directory that
+  // For all top-level or @-scoped folders in the node_modules directory that
   // contain a package.json file...
```

Comment:
Can you provide a bit more detail here about the change for future people looking at the code?  What was it about the deps or structure of the polymer-cli that made this change necessary?

Reply:
I think the behaviour is actually what was originally intended and documented. That is, it's not specific to polymer-cli, it was simply (I think) an oversight in the handling of @-scoped dependencies which meant they were omitted from the generated BUILD.bazel.

I'm happy to add more comments if needed though.